### PR TITLE
Docs: Using postgres and ZFS device migration

### PR DESCRIPTION
**Pull request by nobert**
Addresses #1189

### CODE REVIEW

**Comment by josephglanville** [comment]
LGTM

File: docs/content/faq/how-do-i-use-the-postgres-appliance.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+toc_min_level: 2
+---
+
+# How do I use the Postgres appliance?
+
+Flynn includes a highly available Postgres appliance which it uses for internal services and is available for use by installed apps.
+
+When adding a Postgres resource to an app, Flynn will provision a database and login credentials specifically for that app. That configuration is automatically passed to the app via environment variables:
+
+- `DATABASE_URL` is a Heroku-compatible url with hostname, database name, and login credentials.
```
Comment (lmars) [inline docs/content/faq/how-do-i-use-the-postgres-appliance.md]: Probably worth giving an example value.

File: docs/content/faq/how-do-i-use-the-postgres-appliance.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+title: How do I use the Postgres appliance?
+layout: docs
+toc_min_level: 2
+---
+
+# How do I use the Postgres appliance?
```
Comment (titanous) [inline docs/content/faq/how-do-i-use-the-postgres-appliance.md]: This seems redundant with https://github.com/flynn/flynn/blob/master/docs/content/postgres.html.md

File: docs/content/faq/how-do-i-use-the-postgres-appliance.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+Flynn includes a highly available Postgres appliance which it uses for internal services and is available for use by installed apps.
+
+When adding a Postgres resource to an app, Flynn will provision a database and login credentials specifically for that app. That configuration is automatically passed to the app via environment variables:
```
Comment (lmars) [inline docs/content/faq/how-do-i-use-the-postgres-appliance.md]: We should link to [Adding a database to an app](https://flynn.io/docs/postgres#adding-a-database-to-an-app) here

**Pull request closed without merging**